plugins/ghidra/MUI/build.gradle

```diff
@@ -147,6 +147,12 @@ task uninstallPreviousMUI {
 
 build.dependsOn uninstallPreviousMUI
 
+task copyMUICommonResources(type: Copy) {
+    from "$projectDir/../../common/native_run_settings.json"
+    into "$projectDir/os/linux_x86_64/"
+}
```

Comment:
I think the JSON settings file would make more sense living in the `$projectDir/data` directory because it's not an executable that is OS-specific. Am I missing something about how Ghidra accesses files?

See the skeleton project's description of the directory https://github.com/NationalSecurityAgency/ghidra/tree/da94eb86bd2b89c8b0ab9bd89e9f0dc5a3157055/GhidraBuild/Skeleton/data

Although, I forget exactly how you access those files 🤔

Reply:
Yeah, that should work!

Did a quick google and seems like it should be https://ghidra.re/ghidra_docs/api/ghidra/framework/Application.html#getModuleDataFile(java.lang.String)